quinn-proto/src/config.rs

```diff
@@ -518,12 +519,10 @@ where
     fn default() -> Self {
         let rng = &mut rand::thread_rng();
 
-        let mut token_key = vec![0; S::HmacKey::KEY_LEN];
-        rng.fill_bytes(&mut token_key);
-        Self::new(
-            S::HmacKey::new(&token_key)
-                .expect("HMAC key rejected random bytes; use ServerConfig::new instead"),
-        )
+        let mut master_key = vec![0u8; 64];
```

Comment:
```suggestion
        let mut master_key = [0u8; 64];
```
Since we can use a hardcoded length here, there's no need for a heap allocation.